MapboxMobileEvents/MMEDate.m

```diff
@@ -55,6 +55,8 @@ + (NSDateFormatter *)HTTPDateFormatter {
     if (!_httpDateFormatter) {
         _httpDateFormatter = [[NSDateFormatter alloc] init];
         [_httpDateFormatter setDateFormat:@"E, d MMM yyyy HH:mm:ss Z"];
+        [_httpDateFormatter setTimeZone:[NSTimeZone timeZoneWithName:@"UTC"]];
```

Comment:
@rclee , Updated this formatter to use UTC by default. This means that the Z value will always be +0000 and also allow for test to be consistent.